tests/rptest/scale_tests/franz_go_verifiable_test.py

```diff
@@ -217,3 +221,108 @@ def test_si_with_timeboxed(self, segment_size: int):
         self.redpanda.set_cluster_config(configs, expect_restart=True)
 
         self._workload(segment_size)
+
+
+class FranzGoVerifiableWithSiAndDisruptions(FranzGoVerifiableBase):
+    MSG_SIZE = 100000
+    PRODUCE_COUNT = 20000
```

Comment:
This produce may not be running for long enough for any disruptive actions to happen: 100KiB * 20,000 messages is only 2GB, which is a few seconds of IO when running on dedicated EC2 nodes.

Suggest increasing the size to long enough to get at least a couple of minutes of runtime, and maybe it would also be good for the context decorators to assert that they did at least one action while running, so that people writing tests can't accidentally use them across short time periods + create the illusion that they're testing failures when really they're not.

Reply:
I kept the message size at 100kb after a discussion with @Lazin and @ztlpn where we discussed realistic message sizes in redpanda, but I see the point to ensure some actions are triggered, I will try to increase produce count by a factor of 100 and test locally.

We do have assertion in test that action was triggered [here](https://github.com/redpanda-data/redpanda/pull/4610/files#diff-cc41057b13bceac3a49e8eb9324fd708f401467af5ef6a2095b52a8248c1a8c7R286), I did not put it in the context manager itself to give control to the user. 

But it might be useful to assert on exiting the context (perhaps on the existence of a flag), will explore this.